feature_list/feature_list.cpp

```diff
@@ -0,0 +1,267 @@
+#include "base/logging.hpp"
+#include "base/string_utils.hpp"
+
+#include "coding/file_name_utils.hpp"
+
+#include "geometry/mercator.hpp"
+#include "geometry/point2d.hpp"
+
+#include "indexer/classificator.hpp"
+#include "indexer/classificator_loader.hpp"
+#include "indexer/feature.hpp"
+#include "indexer/feature_processor.hpp"
+#include "indexer/map_object.hpp"
+#include "indexer/map_style_reader.hpp"
+
+#include "platform/local_country_file_utils.hpp"
+#include "platform/platform.hpp"
+
+#include "search/engine.hpp"
+#include "search/reverse_geocoder.hpp"
+#include "search/search_quality/helpers.hpp"
+
+#include "std/iostream.hpp"
+
+#include "storage/index.hpp"
+#include "storage/storage.hpp"
+
+class ClosestPoint
+{
+  m2::PointD const & m_center;
+  m2::PointD m_best;
+  double m_distance = -1;
+
+public:
+  ClosestPoint(m2::PointD const & center) : m_center(center) {}
```

Comment:
m_best is not initialized. This is not good, as default ctor for PointD does not sets fields.